controls/pcidss_4_ocp4.yml

```diff
@@ -89,26 +93,39 @@ controls:
           - Updated as needed upon changes to the environment
       levels:
         - base
-      status: pending
+      # NOTE(rhmdnd): ACS can supplement this requirement through it's
+      # networking graph functionality, which visually depicts the network
+      # flows between resources, namespaces, and the internet. Add a rule here
+      # to check if ACS is available.
+      status: not applicable
 
     - id: 1.2.5
       title: 'All services, protocols, and ports allowed are identified, approved, and have a
         defined business need.'
       levels:
         - base
-      status: pending
+      # NOTE(rhmdnd): This is best left to the system owner to maintain, and
+      # falls outside of the responsibility of OpenShift.
+      status: not applicable
 
     - id: 1.2.6
       title: Security features are defined and implemented for all services, protocols, and ports
         that are in use and considered to be insecure, such that the risk is mitigated.
       levels:
         - base
-      status: pending
+      status: partial
       notes: |-
-        The specific risks associated with the use of insecure services, protocols, and ports are
-        understood, assessed, and appropriately mitigated. The selected rules here basically
-        remove services without encryption and restricted some common services.
-      rules: []
+        The specific risks associated with the use of insecure services,
+        protocols, and ports are understood, assessed, and appropriately
+        mitigated.
+      # NOTE(rhmdnd): Implement a rule to check for a default network policy
+      # for new projects. Good practice would be to have a deny by default
+      # network policy. See the following documentation for how to configure
+      # this in the OpenShift configuration:
+      # https://docs.openshift.com/container-platform/latest/networking/network_policy/default-network-policy.html
+      rules:
+        - configure_network_policies_namespaces
```

Comment:
Need to make sure these rules reference the new requirement/control ID (e.g., 1.2.6) since the requirement IDs from PCI-DSS 3.2.1 don't line up directly with version 4.0.0.

Reply:
I wonder if we need to inspect all services, and also to inspect any pod using hostnetwork?

Reply:
I think network_policies does not apply to pod using hostnetwork